cryptoinvestor/models/User.py

```diff
@@ -0,0 +1,10 @@
+from cryptoinvestor.models.Account import Account
+
+
+class User:
+
+    def __init__(self, *, username="", password="", account_id=None):
```

Comment:
IMO all the arguments here should not have default values, so something like this would be more suiting.

```python
def __init__(self, *, username: str, password: str, account_id: int):
...
```

Reply:
Could we also type the third argument following way ?
```python
def __init__(self, *, username: str, password: str, account: Account):
...
```

Reply:
yes